src/core/client/test/helpers/fixture.ts

```diff
@@ -0,0 +1,339 @@
+import {
+  GQLComment,
+  GQLCOMMENT_STATUS,
+  GQLMODERATION_MODE,
+  GQLSettings,
+  GQLStory,
+  GQLUser,
+  GQLUSER_ROLE,
+  GQLUSER_STATUS,
+} from "coral-framework/schema";
+import {
+  createFixture,
+  denormalizeComment,
+  denormalizeStory,
+} from "coral-framework/testHelpers";
+import uuid from "uuid/v4";
+
+export function createDateInRange(start: Date, end: Date) {
+  return new Date(
+    start.getTime() + Math.random() * (end.getTime() - start.getTime())
+  );
+}
+
+export function randomDate() {
+  return createDateInRange(new Date(2000, 0, 1), new Date());
+}
+
+export function createUserStatus(banned: boolean = false) {
+  return {
+    current: [banned ? GQLUSER_STATUS.BANNED : GQLUSER_STATUS.ACTIVE],
+    ban: {
+      active: banned,
+      history: [],
+    },
+    suspension: {
+      active: false,
+      until: null,
+      history: [],
+    },
+  };
+}
+
+export function createUser() {
+  return createFixture<GQLUser>({
+    id: uuid(),
+    username: uuid(),
+    role: GQLUSER_ROLE.COMMENTER,
+    createdAt: randomDate().toISOString(),
+    status: createUserStatus(),
+    ignoredUsers: [],
+    ignoreable: true,
+  });
+}
+
+export function createComment(author?: GQLUser) {
+  const revision = uuid();
+  const createdAt = randomDate();
```

Comment:
Would suggest adding a TODO relating to fixing the randomness to ensure repeatable tests.

Reply:
Added TODO to the randomDate() in the fixture file.